src/backend/BackendInterface.h

```diff
@@ -274,55 +342,74 @@ class BackendInterface
         std::optional<TransactionsCursor> const& cursorIn,
         boost::asio::yield_context& yield) const = 0;
 
-    // *** state data methods
-    /// Returns specific ledger object, a vector of unsigned characters, given
-    /// unsigned 256-integer representing a key, the specific ledger sequence as
-    /// a standard unsigned 32-bit integer, and the currently executing
-    /// coroutine as input.
+    /*! @brief STATE DATA METHODS */
+    /**
+     * @brief Fetches a specific ledger object: vector of unsigned chars
+     *
+     * @param key Unsigned 256-bit integer.
+     * @param sequence Unsigned 32-bit integer.
+     * @param yield Currently executing coroutine.
+     * @return std::optional<Blob>
+     */
     std::optional<Blob>
     fetchLedgerObject(
         ripple::uint256 const& key,
         std::uint32_t const sequence,
         boost::asio::yield_context& yield) const;
 
-    /// Returns specific ledger object, a vector of vectors of unsigned
-    /// characters, given a vector of unsigned 256-integer each representing
-    /// keys, the specific ledger sequence as a standard unsigned 32-bit
-    /// integer, and the currently executing coroutine as input.
+    /**
+     * @brief Fetches all ledger objects: a vector of vectors of unsigned chars.
+     *
+     * @param keys Unsigned 256-bit integer.
+     * @param sequence Unsigned 32-bit integer.
+     * @param yield Currently executing coroutine.
+     * @return std::vector<Blob>
+     */
     std::vector<Blob>
     fetchLedgerObjects(
         std::vector<ripple::uint256> const& keys,
         std::uint32_t const sequence,
         boost::asio::yield_context& yield) const;
 
-    /// A virtual function version of fetchLedgerObject, taking in the exact
-    /// same parameters.
+    /*! @brief Virtual function version of fetchLedgerObject */
     virtual std::optional<Blob>
     doFetchLedgerObject(
         ripple::uint256 const& key,
         std::uint32_t const sequence,
         boost::asio::yield_context& yield) const = 0;
 
-    /// A virtual function version of fetchLedgerObjects, taking in the exact
-    /// same parameters.
+    /*! @brief Virtual function version of fetchLedgerObjects */
```

Comment:
you don't need to specify "virtual" in the doc comment - the function signature already should contain this information (you need to confirm this is true after generating the doc tho).